src/components/Studio/MyStudio.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+class MyStudio extends Component {
+  render() {
+    return (
+      <div>
+        <div>
+          <h1>{this.props.studioName}</h1>
+          <p>{this.props.street}</p>
+          <p>{this.props.city}</p>
+          <p>{this.props.country}</p>
+          <p>{this.props.email}</p>
+          <p>{this.props.number}</p>
+          <p>{this.props.descriptionOfStudio}</p>
+          <p>{this.props.descriptionOfCaseStudies}</p>
+          <p>{this.props.descriptionOfPractie}</p>
+          <p>{this.props.descriptionOfDesignEducation}</p>
```

Comment:
functional component for sure, as a rule: if it doesn't use local state or lifecycle methods (componentDidMount) then => function.

Another thing, maybe destructure props before the return to make it cleaner?